atc/db/migration/migrations/1570709099_add_varsources_to_pipelines.up.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+  ALTER TABLE pipelines ADD COLUMN var_sources text,
+                        ADD COLUMN var_sources_nonce text;
```

Comment:
Sorry, I shouldn't have suggested this yet; currently the column has to be called `nonce` because our encryption code hardcodes it here:

https://github.com/concourse/concourse/blob/23c12778b8c424f9ed9f009154461b0bd321c755/atc/db/open.go#L136-L139

Also this column will need to be added to the list of encrypted columns here:

https://github.com/concourse/concourse/blob/23c12778b8c424f9ed9f009154461b0bd321c755/atc/db/open.go#L123-L131

This code is for rotating encryption keys on startup.

I'm OK with just calling it `nonce` unless you want to also refactor the encryption code to support other column names. (I doubt you're interested in that since you'd also need to migrate all the existing columns.)